gitprovider/repositoryref.go

```diff
@@ -220,7 +222,7 @@ type UserRepositoryRef struct {
 	RepositoryName string `json:"repositoryName"`
 }
 
-// String returns the HTTPS URL to access the repository.
+// String returns the URL to access the repository.
```

Comment:
Is this true for all implementations, given there were no changes to any of the GitHub logic?

Reply:
True, there are no changes to the Github part, but it might return an `http://` url if it's possible to run github enterprise without https.